cmd/search.go

```diff
@@ -18,17 +18,29 @@ import (
 	"errors"
 	"fmt"
 	"os"
+	"strings"
 
 	"github.com/dropbox/dropbox-sdk-go-unofficial/files"
 	"github.com/spf13/cobra"
 )
 
 func search(cmd *cobra.Command, args []string) (err error) {
-	if len(args) != 1 {
+	if len(args) == 0 {
 		return errors.New("`search` requires a `query` argument")
 	}
 
-	arg := files.NewSearchArg("", args[0])
+	// Parse path scope, if provided.
+	scope := ""
```

Comment:
[nit] Slight preference for `var scope string`; Go uses the value `""` as default for strings.